dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/event/EventOperationParamsMapper.java

```diff
@@ -119,9 +119,16 @@ public EventQueryParams map(
 
     List<Program> accessiblePrograms = getPrograms(program);
 
+    if (queryParams.isSynchronizationQuery()
+        && queryParams.getPsdesWithSkipSyncTrue() != null
+        && !queryParams.getPsdesWithSkipSyncTrue().isEmpty()) {
+      queryParams = queryParams.withSkipSyncFiltering(queryParams.getPsdesWithSkipSyncTrue());
```

Comment:
This statement does not make sense to me. 
You are assigning the value of one property as the value of another property of the same object?